Test/FMI3/fmi3_xml_parsing_test.c

```diff
@@ -53,9 +53,6 @@ void test_parser(char *xml_dir, int should_log_expected_msg, int configuration)
     cb.log_level = jm_log_level_all;
     cb.context   = NULL;
     context = fmi_import_allocate_context(&cb);
-    if (configuration != 0) {
-        fmi_import_set_configuration(context, configuration);
-    }
 
     g_has_logged_expected_msg = 0;
 
```

Comment:
Is g_has_logged_expected_msg set by fmi3_import_parse_xml?

Reply:
See other comment, it is set via `importlogger`.

Reply:
Thanks, I think I understand how this works now.